.github/workflows/sync-template.yml

```diff
@@ -3,7 +3,7 @@ name: Sync App Template Repository
 on:
   workflow_dispatch: {}
   push:
-    branches: [main]
+    branches: [sdk-51]
```

Comment:
Are we now going to update this as part of each release? Can't we do `sdk-*` here and then make a simple condition to skip steps when the SDK branch is not the latest?

Also, shouldn't we version the template, i.e. have equivalent branches for each SDK?

Reply:
> Are we now going to update this as part of each release? Can't we do sdk-* here and then make a simple condition to skip steps when the SDK branch is not the latest?

this is a good idea. we should do this

> Also, shouldn't we version the template, i.e. have equivalent branches for each SDK?

this isn't required -- the only purpose of this template sync is for expo.new, where we always want to use the latest version of the template for the latest sdk release

Reply:
Good point about `sdk-*`. What's the best way to check if SDK is latest in that context?

---

> the only purpose of this template sync is for expo.new, where we always want to use the latest version of the template for the latest sdk release

It wouldn't hurt though to give users more options in future, similar to how versioning used to work in Expo Go (but I might be missing something and maybe it doesn't make practical sense).

Reply:
the canonical way is to check https://exp.host/--/api/v2/versions for the most recent version with a `releaseNoteUrl` 👀 or alternatively check which version has latest tag on npm

Reply:
The latter is probably the easiest and more reliable, so something like `npm view expo@latest version --silent | cut -d . -f1` to get the number.